client/src/components/Event.jsx

```diff
@@ -88,11 +109,23 @@ class Event extends React.Component {
 
     if (this.state.hasPermissionToView) {
       const { id } = this.props.match.params;
-      const { title, description } = this.state;
+      const { title, description, delivery_time } = this.state;
+
+      let LaunchTimeLen = moment(delivery_time).length;
+      let launchTime = moment(this.state.delivery_time).format('YYYY MM DD hh mm ss');

```

Comment:
no need to type `this.state.delivery_time` and can reference `delivery_time` due to your declaration on `line 112`